plat/mediatek/mt6795/include/mcucfg.h

```diff
@@ -104,12 +104,10 @@ struct mt6795_mcucfg_regs {
 static struct mt6795_mcucfg_regs *const mt6795_mcucfg = (void *)MCUCFG_BASE;
 
 /* cpu boot mode */
-enum {
-	MP0_CPUCFG_64BIT_SHIFT = 12,
-	MP1_CPUCFG_64BIT_SHIFT = 28,
-	MP0_CPUCFG_64BIT = 0xf << MP0_CPUCFG_64BIT_SHIFT,
-	MP1_CPUCFG_64BIT = 0xf << MP1_CPUCFG_64BIT_SHIFT
-};
+#define	MP0_CPUCFG_64BIT_SHIFT	12
+#define	MP1_CPUCFG_64BIT_SHIFT	28
+#define	MP0_CPUCFG_64BIT	(0xfU << MP0_CPUCFG_64BIT_SHIFT)
+#define	MP1_CPUCFG_64BIT	(0xfU << MP1_CPUCFG_64BIT_SHIFT)
```

Comment:
minor: Given that PR #953 adds a new `U(_x)` macro, it might be worth waiting until that is merged and use that here, although that is not mandatory as the code is not used in a linker/assembly environment.